test/kubernetes_unittest.cc

```diff
@@ -13,6 +15,16 @@ class KubernetesTest : public ::testing::Test {
       throw(json::Exception) {
     return reader.GetNodeMetadata(node, collected_at, is_deleted);
   }
+
+  json::value ComputePodAssociations(const json::Object* pod,
+				     const KubernetesReader& reader) {
```

Comment:
Indentation. Want to switch your editor config to expand tabs to spaces? Also below.